Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -93,10 +93,15 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
 
   // options
   groups.emplace_back(m_options = new ControllerEmu::ControlGroup(_trans("Options")));
-  m_options->AddSetting(&m_always_connected_setting,
-                        // i18n: Treat a controller as always being connected regardless of what
-                        // devices the user actually has plugged in
-                        _trans("Always Connected"), false);
+  m_options->AddSetting(
+      &m_always_connected_setting,
+      // i18n: Treat a controller as always being connected regardless of what
+      // devices the user actually has plugged in
+      {_trans("Always Connected"), _trans(""),
+       _trans("Always connected if checked.\nIf unchecked, it will link the emulated controller "
+              "connection state\nto the "
+              "real default device connection state (if there is one).")},
```

Comment:
Minor, but I think this would look better like this (it's the same number of lines, but it more closely reflects what's actually shown):

```
       _trans("Always connected if checked.\n"
              "If unchecked, it will link the emulated controller connection state\n"
              "to the real default device connection state (if there is one).")},
```

Reply:
True, I will change it, but if we are gonna worry about these nitpick, we are never gonna get my other main PRs merged 😅

Reply:
Not necessarily, I think that with this change, all natural languages segments of the diff are accounted for.